tests/h/services/search_index/_queue_test.py

```diff
@@ -148,36 +155,28 @@ def test_it_does_nothing_if_the_queue_is_empty(self, batch_indexer, queue):
         batch_indexer.index.assert_not_called()
 
     def test_it_ignores_jobs_that_arent_scheduled_yet(
-        self, annotation_ids, batch_indexer, queue
+        self, add_all, batch_indexer, queue
     ):
-        queue.add_all(annotation_ids, tag="test_tag", schedule_in=ONE_WEEK_IN_SECONDS)
+        add_all(schedule_in=ONE_WEEK_IN_SECONDS)
 
         queue.sync(LIMIT)
 
         batch_indexer.index.assert_not_called()
 
+    @pytest.mark.usefixtures("with_queued_annotations")
     def test_it_ignores_jobs_beyond_limit(
         self, all_annotation_ids, batch_indexer, queue
     ):
-        queue.add_all(
-            all_annotation_ids, tag="test_tag", schedule_in=MINUS_5_MIN_IN_SECS
-        )
-
         queue.sync(LIMIT)
 
         for annotation_id in all_annotation_ids[LIMIT:]:
             assert annotation_id not in batch_indexer.index.call_args[0][0]
 
+    @pytest.mark.usefixtures("with_indexed_annotations")
     def test_if_the_job_has_force_True_it_indexes_the_annotation_and_deletes_the_job(
-        self, annotations, annotation_ids, batch_indexer, db_session, index, queue, LOG
+        self, annotation_ids, add_all, batch_indexer, db_session, queue, LOG
     ):
-        index(annotations)
-        queue.add_all(
-            annotation_ids,
-            tag="test_tag",
-            schedule_in=MINUS_5_MIN_IN_SECS,
-            force=True,
-        )
+        add_all(force=True)
```

Comment:
And of course, no later than the next test we see something where we can't use `with_queued_annotations` due to the order. I still prefer this anyway, as it shows us the important bit (force in this case)